lighthouse-cli/test/smokehouse/readme.md

```diff
@@ -64,11 +64,14 @@ Individual elements of an array can be asserted by using numeric properties in a
 
 However, if an array literal is used as the expectation, an extra condition is enforced that the actual array _must_ have the same length as the provided expected array.
 
+Arrays can be checked against a subset of elements using the special `_includes` property. The value of `_includes` _must_ be an array.
```

Comment:
What did you think of naming this `_subarray` or `_subset`? For example, if another `_` helper were added that didn't exclude items as it checked each item, what would the two helpers be named to distinguish them from each other?

should add a note that each assertion in `_include` will exclude the matching item from consideration in the rest

Reply:
> if another _ helper were added that didn't exclude items as it checked each item, what would the two helpers be named to distinguish them from each other?

I'm ok with `_subset`, but I can't think of a situation where I would use a helper that doesn't exclude as it goes.

Reply:
OK. let's just use `_includes` for now then.